controller/article.controller.js

```diff
@@ -0,0 +1,13 @@
+const { selectArticlesById } = require("../model/article-model");
+
+exports.getArticlesById = (req, res, next) => {
+  const { article_id } = req.params;
```

Comment:
Great use of object destructuring here to get article_id from the request params :)